app/assets/javascripts/discourse/app/controllers/upload-selector.js

```diff
@@ -1,49 +1,51 @@
 import {
   allowsAttachments,
   authorizedExtensions,
-  authorizesAllExtensions,
   uploadIcon,
 } from "discourse/lib/uploads";
 import Controller from "@ember/controller";
 import I18n from "I18n";
 import ModalFunctionality from "discourse/mixins/modal-functionality";
 import discourseComputed from "discourse-common/utils/decorators";
-import { equal } from "@ember/object/computed";
+import { alias, equal } from "@ember/object/computed";
 
 export default Controller.extend(ModalFunctionality, {
   imageUrl: null,
   local: equal("selection", "local"),
   remote: equal("selection", "remote"),
   selection: "local",
+  showFormatsHint: alias("allowAdditionalFormats"),
```

Comment:
are you sure you need alias here ?

Reply:
Wasn't sure about it :) I'd better remove it then.